src/observable/dom/AjaxObservable.ts

```diff
@@ -304,14 +303,14 @@ export class AjaxSubscriber<T> extends Subscriber<Event> {
     (<any>xhrTimeout).request = request;
     (<any>xhrTimeout).subscriber = this;
     (<any>xhrTimeout).progressSubscriber = progressSubscriber;
-    if (xhr.upload && 'withCredentials' in xhr && root.XDomainRequest) {
+    if (xhr.upload && 'withCredentials' in xhr) {
```

Comment:
I think `&&` constraint is probably needed to be updated to deal each cases (`if xhr` vs `else if XDomainRequest`), but checking itself is still needed.